sqlparser/lib/utils/node_to_text.dart

```diff
@@ -727,7 +727,6 @@ class NodeSqlBuilder extends AstVisitor<void, void> {
         InsertMode.insertOrFail: TokenType.fail,
         InsertMode.insertOrIgnore: TokenType.ignore,
       }[mode]!);
-      visitNullable(e.returning, arg);
```

Comment:
Well this is embarrassing :sweat_smile: Thanks for the fix! Could you also add a test case similar to this one? https://github.com/simolus3/drift/blob/9e4e177b97d9804aa7bc903b42ad07d4dca2b8fb/sqlparser/test/utils/node_to_text_test.dart#L303-L305

Reply:
I added the test case you ask for and also edited the tests for nested queries to include variables that refer to the enclosing query :)